docs/source/tutorials/oregon.rst

```diff
@@ -0,0 +1,625 @@
+    for stratum, results in individual_results.items():
+        if results is None or col_idx > 2:
+            continue
+
+        plot(results['locations'], results['ml']['ldte'],
+                results['ml']['lower'], results['ml']['upper'],
+                title=f"ED Costs: {stratum}\n(ML-Adjusted Local Estimator, n={results['sample_size']:,})",
+                xlabel="Emergency Department Costs",
+                ylabel="Local Distribution Treatment Effect",
+                color="red" if col_idx == 1 else "purple", ax=axes[1, col_idx])
+        col_idx += 1
+
+    plt.suptitle("Comparison: Overall Population vs Individual Household Registration Strata (Local Estimators)", fontsize=16)
+    plt.tight_layout()
+    plt.show()
+
+.. image:: ../_static/oregon_ldte_costs_strata.png
+   :alt: Oregon Health Insurance Experiment DTE Strata Analysis
+   :width: 800px
+   :align: center
+
+**1. Overall Population vs Stratified Analysis**
+
+- **Overall Population (Left panels)**:
+
+  - Simple: LDTE ≈ -0.21 at zero costs, converging to zero around $10,000
+  - ML-Adjusted: LDTE ≈ -0.15 at zero costs, similar convergence pattern
+  - Both show consistent rightward distribution shifts across the entire population
+
+- **Signed Self Up (Middle panels, n=12,982)**:
+
+  - Simple: LDTE ≈ -0.18 at zero costs, converging to zero around $20,000
+  - ML-Adjusted: LDTE ≈ -0.20 at zero costs, similar pattern
+  - Smaller magnitude effects compared to overall population, suggesting this stratum has more moderate responses to insurance
+
+- **Signed Self Up + Others (Right panels, n=4,068)**:
+
+  - Simple: LDTE ≈ -0.55 at zero costs, converging to zero around $15,000-$20,000
+  - ML-Adjusted: Shows extreme values (≈ -0.30 to +20 near zero costs with very wide confidence intervals)
+  - Much larger magnitude effects, indicating households with multiple members show substantially stronger treatment effects
+
+**2. Heterogeneity Across Strata**
+
+The stratified analysis reveals substantial treatment effect heterogeneity:
+
+- **"Signed self up" stratum**: Moderate effects (LDTE ≈ -0.18 to -0.20), suggesting single-person households have more modest increases in ED utilization
+- **"Signed self up + others" stratum**: Large effects (LDTE ≈ -0.55 for Simple), suggesting multi-person households experience much greater increases in ED access
+- The 3-4x larger effect in the "signed self up + others" group indicates that household composition is a critical moderator of insurance impact
+
+**3. Estimation Challenges and Confidence Intervals**
+
+- **Overall population**: Both estimators show reasonable confidence intervals, with ML adjustment providing modest improvements in the mid-range.
+- **"Signed self up" stratum**: Confidence intervals remain wide but manageable for both estimators, showing similar patterns to the overall population.
+- **"Signed self up + others" stratum**:
+
+  - Extreme estimation instability, particularly for ML-adjusted estimator
```

Comment:
The ML estimation is a bit odd, have you tried using different ml models or fold number? We didn't face this issue for the Neurips paper.